File: sistemastreamingruby/usuario.rb
class Usuario
    attr_accessor :nombre, :edad, :suscripcion
  
    def initialize(nombre, edad, suscripcion)
      @nombre = nombre
      @edad = edad
      @suscripcion = suscripcion
    end
  
    def to_s
      "Nombre: #{@nombre}, Edad: #{@edad}, Suscripción: #{@suscripcion}"
    end
  end
  

File: sistemastreamingjs/usuario.js
class Usuario {
    constructor(nombre, edad, suscripcion) {
        this.nombre = nombre;
        this.edad = edad;
        this.suscripcion = suscripcion;
    }

    toString() {
        return `Nombre: ${this.nombre}, Edad: ${this.edad}, Suscripción: ${this.suscripcion}`;
    }
}

module.exports = Usuario;
